stack.yaml

```diff
@@ -50,13 +60,105 @@ extra-deps:
 
-nix:
-  shell-file: stack-deps.nix
+- git: https://github.com/kim/snappy-framing
+  commit: d99f702c0086729efd6848dea8a01e5266c3a61c
+
+- git: https://gitlab.com/twittner/wai-routing
+  commit: 7e996a93fec5901767f845a50316b3c18e51a61d
+
+# Includes the changes from <https://gitlab.com/twittner/cql-io/merge_requests/14>
+# - git: https://gitlab.com/twittner/cql-io.git
+#   commit: 8b91d053c469887a427e8c075cef43139fa189c4
+
+#  Our fork of multihash with relaxed upper bounds
+- git: https://github.com/wireapp/haskell-multihash.git
+  commit: 300a6f46384bfca33e545c8bab52ef3717452d12
+
+# Our fork of aws with minor fixes
+- git: https://github.com/wireapp/aws
+  commit: 42695688fc20f80bf89cec845c57403954aab0a2
+
+# https://github.com/hspec/hspec-wai/pull/49
+- git: https://github.com/wireapp/hspec-wai
+  commit: 0a5142cd3ba48116ff059c041348b817fb7bdb25
+
+# amazonka-1.6.1 is buggy: https://github.com/brendanhay/amazonka/issues/466
+# Therefore we pin an unreleased commit directly.
+#
+# More precisely, we pull just some libraries out of it,
+# the other packages weren't changed between 1.6.1 and this commit,
+# so we can use Stackage-supplied versions for them.
+# See https://github.com/brendanhay/amazonka/compare/1.6.1...9cf5b5777b69ac494d23d43a692294882927df34
+#
+# Once there has been made a new hackage release, we can use that instead.
+- archive: https://github.com/brendanhay/amazonka/archive/9cf5b5777b69ac494d23d43a692294882927df34.tar.gz
+  sha256: c3044f803a7652aee88fe600a97321175cdc1443d671246ba7ff78e14bf5b49f
+  size: 11137527
+  subdirs:
+  - amazonka
+  - amazonka-elb
+  - amazonka-redshift
+  - amazonka-route53
+  - core
+
+############################################################
+# Wire packages
+############################################################
+
+- git: https://github.com/wireapp/cryptobox-haskell
+  commit: 7546a1a25635ef65183e3d44c1052285e8401608    # master (Jul 21, 2016)
+
+- git: https://github.com/wireapp/hsaml2
+  commit: fe08618e81dee9b7a25f10f5b9d26d1ff1837c79    # master (Mar 25, 2020)
+
+- git: https://github.com/wireapp/http-client
+  commit: a160cef95d9daaff7d9cfe616d95754c2f8202bf    # master (Feb 4, 2020)
+  subdirs:
+  - http-client
+  - http-client-openssl
+  - http-client-tls
+  - http-conduit
+
+# Dropped from upstream snapshot
+- bloodhound-0.16.0.0
+- template-0.2.0.10
+- HaskellNet-0.5.1
+- HaskellNet-SSL-0.3.4.1
+- snappy-0.2.0.2
+- smtp-mail-0.2.0.0
+- stm-containers-1.1.0.4
+- redis-io-1.0.0
+- redis-resp-1.0.0
+- hedgehog-quickcheck-0.1.1
+
+# Only in nightly
+- stm-hamt-1.2.0.4
+- optics-th-0.2
+- primitive-unlifted-0.1.2.0
+
+# Not on stackage
+- currency-codes-3.0.0.1
+- mime-0.4.0.2
+- data-timeout-0.3.1
+- geoip2-0.4.0.1
+- stomp-queue-0.3.1
+- text-icu-translit-0.1.0.7
+- wai-middleware-gunzip-0.0.2
+- cql-io-tinylog-0.1.0
+- invertible-hxt-0.1
+- network-uri-static-0.1.2.1
+- base58-bytestring-0.1.0
+- stompl-0.5.0
+- pattern-trie-0.1.0
+
+# Not latest as latst one breaks wai-routing
```

Comment:
```suggestion
# Not latest as latest one breaks wai-routing
```

Reply:
it at least seems to work on CI; which is promising!  "Using precompiled package" messages everywhere